auto_show/admin.py

```diff
@@ -1,3 +1,28 @@
 from django.contrib import admin
 
-# Register your models here.
+from auto_show.models import AutoShow, ActionAutoShow, DiscountAutoShow
+
+
+class AutoShowAdmin(admin.ModelAdmin):
+    """Класс для админки модели AutoShow"""
+
+    list_display = ('name', 'country', 'balance', 'wish_cars', 'list_auto', 'is_active')
+    list_editable = ('is_active',)
+
+class DiscountAutoShowAdmin(admin.ModelAdmin):
+    """Класс для админки модели DiscountAutoShow"""
+
+    list_display = ('auto_shows', 'name', 'amount_discount',  'min_amount_spent', 'max_amount_spent', 'is_active')
+    list_editable = ('is_active',)
+
+
+class ActionAutoShowAdmin(admin.ModelAdmin):
+    """Класс для админки модели ActionAutoShow"""
+
+    list_display = ('name', 'amount_action', 'date_start', 'date_finish', 'auto_shows', 'is_active')
+    list_editable = ('is_active',)
+
+
+admin.site.register(AutoShow, AutoShowAdmin)
+admin.site.register(ActionAutoShow, ActionAutoShowAdmin)
+admin.site.register(DiscountAutoShow, DiscountAutoShowAdmin)
```

Comment:
don't forget about blank line at the end of a file

Reply:
+